xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemuxClient.cpp

```diff
@@ -427,6 +431,13 @@ void CDVDDemuxClient::RequestStreams()
       st->iBlockAlign     = source->iBlockAlign;
       st->iBitRate        = source->iBitRate;
       st->iBitsPerSample  = source->iBitsPerSample;
+      if (source->ExtraSize > 0 && source->ExtraData)
```

Comment:
I am not a friend of creating a method for 3 lines of code that is only used 2 times in the entire code.